src/tictactoe_clojure/player.clj

```diff
@@ -0,0 +1,25 @@
+(ns tictactoe-clojure.player
+  (:require [tictactoe-clojure.board :as board])
+  (:require [tictactoe-clojure.human-player :as human-player])
+  (:require [tictactoe-clojure.computer-player :as computer-player]))
+
+(defprotocol Player
+  (piece [this])
+  (move [this board]))
+
+(defrecord Human [piece]
+  Player
+  (piece [this] (:piece this))
+  (move [this board]
+    (human-player/human-move board)))
+
+(defrecord Computer [piece]
+  Player
+  (piece [this] (:piece this))
+  (move [this board]
+    (computer-player/optimal-move (:piece this) board 9)))
```

Comment:
I was passing in the depth for a 3x3 board. But I removed passing that in as a parameter in PR#9. In PR#9 the board depth is being calculated.